appeals/e2e/cypress/e2e/back-office-appeals/unlinkAppeals.spec.js

```diff
@@ -0,0 +1,41 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { CaseDetailsPage } from '../../page_objects/caseDetailsPage.js';
+import { ListCasesPage } from '../../page_objects/listCasesPage';
+import { urlPaths } from '../../support/urlPaths';
+import { tag } from '../../support/tag';
+import { happyPathHelper } from '../../support/happyPathHelper.js';
+
+const listCasesPage = new ListCasesPage();
```

Comment:
This doesn't appear to me in use, so could be removed

Reply:
Good point - thanks, I copied it over from linkedAppeals  but don't use it unlinked Appeals.

Reply:
happy path helper is used though